pkg/tnf/handlers/liveness/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2020-2022 Red Hat, Inc.
```

Comment:
your code does not actually test anything. 
you should create the tests in lifecycle/suite.go